docs/dev/development.adoc

```diff
@@ -448,15 +448,13 @@ $ git push upstream v0.0.1
 ----
 *Or* create the new release using the GitHub site (this must be a proper release and not a draft).
 +
-NOTE: Do not upload any binaries for the release. When the new tag is created, Travis-CI starts a special deploy job. This job builds the binaries automatically (using `make prepare-release`) and then uploads it to the GitHub release page. When the job finishes you should see the binaries on the GitHub release page. The release is now marked as a draft.
 
 . Update the descriptions and publish the release.
 . Verify that packages have been uploaded to the `rpm` and `deb` repositories.
 . Update the Homebrew package:
 .. Check commit id for the released tag `git show-ref v0.0.1`
 .. Create a PR to update `:tag` and `:revision` in the https://github.com/kadel/homebrew-odo/blob/master/Formula/odo.rb[`odo.rb`] file
 in https://github.com/kadel/homebrew-odo[`kadel/homebrew-odo`].
-. Confirm the binaries are available in the GitHub release page.
```

Comment:
I am a bit confused but don't we need to confirm that the binaries are there before the next step?

Reply:
Yeah, we no longer build binaries on travis / GitHub. It's all synced on the internal OpenShift CI.